docs/langgraph_checkpoint.ipynb

```diff
@@ -0,0 +1,521 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "dqWWKcgj0vz1"
+      },
+      "outputs": [],
+      "source": [
+        "# Replace with your own AlloyDB Omni info\n",
+        "connstring = f\"postgresql+asyncpg://{OMNI_USER}:{OMNI_PASSWORD}@{OMNI_HOST}:{OMNI_PORT}/{OMNI_DATABASE}\"\n",
+        "engine = AlloyDBEngine.from_engine_args(connstring)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "eyh2vCQq0vz1"
+      },
+      "source": [
+        "### Initialize a table\n",
+        "The `AlloyDBSaver` class requires a database table with a specific schema in order to store the persist LangGraph agents state.\n",
+        "\n",
+        "The `AlloyDBEngine` engine has a helper method `init_checkpoint_table()` that can be used to create a table with the proper schema for you."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ehpAbwH50vz1"
+      },
+      "outputs": [],
+      "source": [
+        "engine.init_checkpoint_table(table_name=TABLE_NAME)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qdIspizq0vz1"
+      },
+      "source": [
+        "#### Optional Tip: 💡\n",
+        "You can also specify a schema name by passing `schema_name` wherever you pass `table_name`. Eg:\n",
+        "\n",
+        "```python\n",
+        "SCHEMA_NAME=\"my_schema\"\n",
+        "\n",
+        "engine.init_checkpoint_table(\n",
+        "    table_name=TABLE_NAME,\n",
+        "    schema_name=SCHEMA_NAME     # Default: \"public\"\n",
+        ")\n",
+        "```"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "a-UERu8q0vz2"
+      },
+      "source": [
+        "### AlloyDBSaver\n",
+        "\n",
+        "To initialize the `AlloyDBSaver` class you need to provide only 4 things:\n",
+        "\n",
+        "1. `engine` - An instance of a `AlloyDBEngine` engine.\n",
+        "1. `table_name` : Table name that stores the checkpoints (default: \"checkpoints\").\n",
+        "1. `schema_name` : The schema name where the table is located (default: \"public\").\n",
+        "1. `serde`: Serializer for encoding/decoding checkpoints (default: None)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "6pr-ZJR-0vz2"
+      },
+      "outputs": [],
+      "source": [
+        "from langchain_google_alloydb_pg import AlloyDBSaver\n",
+        "\n",
+        "checkpointer = AlloyDBSaver.create_sync(\n",
+        "    engine,\n",
+        "    table_name=TABLE_NAME,\n",
+        "    # schema_name=SCHEMA_NAME,\n",
+        "    # serde=None,\n",
+        ")\n",
+        "\n",
```

Comment:
Can you split this into another code section with a  header "Example of Checkpointer methods"?